modules/js/src/loader.js

```diff
@@ -0,0 +1,108 @@
+function Loader() {
+    let self = this;
+    let asmPath = "";
+    let wasmPath = "";
+    let simdPath = "";
+    let threadsPath = "";
+    let mtSIMDPath = "";
+    
+    
+    this.judgeWASM = function() {
+        try{
+            let test = WebAssembly;
+            return true;
+        } catch(e) {
+            return false;
+        }
+    }
+
+    this.judgeSIMD = function() {
+        return WebAssembly.validate(new Uint8Array([0,97,115,109,1,0,0,0,1,4,1,96,0,0,3,2,1,0,10,9,1,7,0,65,0,253,15,26,11]));
```

Comment:
could you please document what's the content of the `Unit8Array`?

Reply:
and you need to call `checkWasm` first.

Reply:
By reading the docs of [wasm-feature-detect](https://github.com/GoogleChromeLabs/wasm-feature-detect), I think my implementation for detecting threads feature and simd feature is good enough. Because the browser will update in the future, which may block my implementation. So, I decide to utilize the [wasm-feature-detect](https://github.com/GoogleChromeLabs/wasm-feature-detect) to solve my problem.

Reply:
It sounds good to me.